back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/importers/idea_importer.rb

```diff
@@ -56,8 +56,6 @@ def add_body_multiloc(idea_row, idea_attributes)
     end
 
     def add_project(idea_row, idea_attributes)
-      raise Error.new 'bulk_import_ideas_blank_project', row: idea_row[:id] if idea_row[:project_title].blank? && idea_row[:project_id].blank?
-
```

Comment:
The way this works now, this should never happen as it's added from the current project so removed the error state